Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Multiplayer Use cases - Unity 6 update

### Description
- Updated Multiplayer Use Cases sample to Unity 6
- Added TextMeshPro essentials
- Updated materials to most recent URP
- Removed obsolete packages
- Replace ParrelSync with Multiplayer Play Mode 

NOTE: this PR is targeting `main` but it should target `develop`. However develop is currently behind `main` because of merge conflicts with the Distributed Authority sample, so we'll need to fix that first, then switch the target branch in this PR.

### Issue Number(s)
- https://jira.unity3d.com/browse/MTT-7917
- https://jira.unity3d.com/browse/MTT-7918
- https://jira.unity3d.com/browse/MTT-8384
- https://jira.unity3d.com/browse/MTT-8707

### Contribution checklist

### How to validate
1. Clone this branch
2. Open the project with Unity 6000.0.17f1
3. Check that no errors originate from the project's scripts
4. Open the tutorials window from Tutorials > Show tutorials
5. Go through all tutorials and check that they can be completed
